scripts/azure/storage-manager.sh

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+readonly CREATE_ACTION="create"
+readonly DELETE_ACTION="delete"
+
+log_info() {
```

Comment:
Did you consider using the log functions in `bin/common.sh`? Thar would make further alternations to logs be more uniform :slightly_smiling_face:

Reply:
I will check it out, i basically used the s3 logging scripts.